pbf/models.py

```diff
@@ -247,6 +247,12 @@ class Meta:
     ROT_GAINED_THIS_TURN = 1 << 2 # Whether they've gained from their track (NOT incrementing using the +1 button)
     ROT_CONVERTED_THIS_TURN = 1 << 3
 
+    def spirit_specific_incremented_this_turn(self):
```

Comment:
no strong opinion on whether to use the generic names here, or ones specific to unconstrained fangs `beasts_prepared_this_turn` and `beasts_added_this_turn`